composer.json

```diff
@@ -27,13 +27,42 @@
         "illuminate/events": "5.7.*",
         "illuminate/hashing": "5.7.*",
         "illuminate/routing": "5.7.*",
-        "twig/twig": "^2.4"
+        "twig/twig": "^2.4",
```

Comment:
This seems like a lot of extra dependencies just to add PHPStan. I would have expected there to only be changes to `require-dev` and that should just be:

```json
"phpstan/phpstan": "^0.11.12"
```